src/components/Image/index.tsx

```diff
@@ -59,12 +61,14 @@ function Image({source: propsSource, isAuthTokenRequired = false, session, onLoa
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [propsSource, isAuthTokenRequired]);
 
+    const shouldOpacityBeZero = isObjectPositionTop && !aspectRatio;
+
     return (
         <BaseImage
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...forwardedProps}
             onLoad={handleLoad}
-            style={[style, aspectRatio ? {aspectRatio, height: 'auto'} : {}, isObjectPositionTop && !aspectRatio && {opacity: 0}]}
+            style={[style, !doNotSetAspectRatioInStyle && aspectRatio ? {aspectRatio, height: 'auto'} : {}, shouldOpacityBeZero && {opacity: 0}]}
```

Comment:
I think it's a little cleaner to rename this to `shouldSetAspectRatio`.

Reply:
Yes sorry I wasn't clearer, it's the `doNot...` part that I took issue with. Affirmative names keeps us in line with the rest of the codebase.